math_hurdler.py

```diff
@@ -113,11 +119,12 @@ def get_sound_path(self, sound_name):
         return os.path.join('./assets/sounds', sound_name)
 
     def write_file(self):
-        return self.points, self.playing
+        return self.points, self.hscore, self.playing
 
-    def restore_game(self, score = '', play_state = ''):
+    def restore_game(self, score = '', hscore = '', play_state = ''):
```

Comment:
Could you remove the spaces before and after the `=` sign as these are keyword args and they shouldn't be any spaces there.